### PR TITLE
[CAM-11117] Cloud Academy: Reviewing documentation regarding version control and auto-updates

### PR DESCRIPTION
**Pull request by SergioQCL**
@gsanchezu please take a look

### CODE REVIEW

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 1]
```diff
@@ -1,82 +1,102 @@
...
+An active Cloud Application Manager account.
+
+### Introduction
+
+In large-scale, complex deployments where tens or hundreds of distributed applications run from a single box configuration, a way of managing changes to box configurations and handling those updates to running instances is needed. Versioning and auto-updates can help with that.
 
 ### Versioning a Box
 
-A box must be versioned to push its updates automatically to live instances deployed from it. To create a version, navigate to the box and click the Versions tab.
+A box must be versioned to push its updates automatically to live instances deployed from it. To create a version, navigate to the box and click the **Versions** tab.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]:
```suggestion
A box must be versioned to push its updates, either manually or automatically, to live instances deployed from it. To create a version, navigate to the box and click the **Versions** tab.
```

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 2]
```diff
@@ -1,82 +1,102 @@
...
 
 ![version-control-and-auto-updates-1.png](../../images/cloud-application-manager/version-control-and-auto-updates-1.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]: Use meaningful alternate texts

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 3]
```diff
@@ -1,82 +1,102 @@
...
 
 **Draft, current, latest**
 
-* Latest is the highest version number of a box. When you deploy from the Instances page, you always get the latest.
-* Draft indicates changes not yet saved to the box. When you deploy from the box page, you typically deploy the draft.
-* Current refers to a specific version restored to the box. If you want to restore a box version you have to do it from the versions tab. Select the gear icon and click on Restore.
+* Latest is the highest version number of a box. When you deploy from Instances page, you always get the latest version.
+* Draft indicates changes not yet saved to the box. When you deploy from the box page, you typically deploy the draft version.
+* Current refers to a specific version restored to the box. If you want to restore a box version you have to do it from the **Versions** tab. Select the gear icon and click on **Restore**.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]:
```suggestion
* Current refers to a specific restored version of the box. If you want to restore a box version you have to do it from the **Versions** tab. Select the gear icon and click on **Restore**.
```

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 4]
```diff
@@ -1,82 +1,102 @@
...
 
 ![version-control-and-auto-updates-3.png](../../images/cloud-application-manager/version-control-and-auto-updates-3.png)
 
 ### Updating Instances Automatically or Manually
 
-When launching an instance, choose the level of auto update you’re comfortable applying. Go with **All Updates** if you are not concerned about big or small configuration changes. Otherwise, choose **Minor and Patch Updates** or **Patch Updates**, which make sure the configuration remains unchanged, for the most part.
+When launching an instance, choose the level of auto-update you’re comfortable applying. Go with **All Updates** if you are not concerned about big or small configuration changes. Otherwise, choose **Minor and Patch Updates** or **Patch Updates**, which make sure the configuration remains unchanged, for the most part.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]: Please, include what will happen, something like (validate this behavior):
```suggestion
When a new version of the box is created, the instances with eligible for automatic update (if the type of version created matches the auto-update policy chosen at deployment time) will be updated with the new version of the box, and a reconfigure action will be automatically triggered to apply the new version changes.
```

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 5]
```diff
@@ -1,82 +1,102 @@
...
 ![version-control-and-auto-updates-7.png](../../images/cloud-application-manager/version-control-and-auto-updates-7.png)
 
-**Note:** When deploying a box from the Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the Versions tab > gear icon, click **Deploy** on a select version.
+**Note:** When deploying a box from Instances page, you consume the latest configuration of the parent and child boxes unless you chose a specific version for the child boxes. To deploy a different version of the parent box, launch it from the box page. From the **Versions** tab > gear icon, click **Deploy** on a selected version.
 
-### Manage a version box
+### Managing box versions
 
-Version boxes can be managed in different ways. To see these options, go to versions tab and select gear icon.
+Box versions can be managed in different ways. To see these options, go to **Versions** tab and click the gear icon.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]:
```suggestion
Box versions can be managed in different ways. To see these options, go to **Versions** tab in the box details page and click the gear icon.
```

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 6]
```diff
@@ -1,82 +1,102 @@
...
 
 ![version-control-and-auto-updates-8.png](../../images/cloud-application-manager/version-control-and-auto-updates-8.png)
 
-1. **Delete**: You can delete the selected version box. Please be aware that clicking on this option deletes the version without a warning popup.
-2. **Deploy**: You can deploy the selected version box.
-3. **Diff**: You can see the updates made to each version box from the versions tab. Select gear icon and click on **Diff**.
+1. **Delete**: You can delete the selected box version. Please be aware that clicking on this option deletes the version without a warning popup.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]: Is this still true? Maybe a warning message was added...

File: Cloud Application Manager/Automating Deployments/version-control.md [hunk 7]
```diff
@@ -1,82 +1,102 @@
...
+2. **Deploy**: You can deploy the selected box version.
+3. **Diff**: You can see the updates made to each box version from the **Versions** tab. Select the gear icon and click on **Diff**.
 
 ![version-control-and-auto-updates-9.png](../../images/cloud-application-manager/version-control-and-auto-updates-9.png)
 
-4. **Restore**: You can restore your version box to its previous version. The version you are working on now will turn into the current version.
-5. **Publish box**: This option changes the privacy settings of the version box, making it public, after being published.  To execute this change, complete the request form. Please note that only those fields marked with an asterisk are mandatory.
+4. **Restore**: You can restore your box to its previous version. The version you are working on now will become the current version.
+5. **Publish box**: This option changes the privacy settings of the box, making it public after being published. To execute this change filling a request form is necessary. Please note that only those fields marked with an asterisk are mandatory.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/version-control.md]: Please, link to an article explaining the publish box feature

File: Cloud Application Manager/Automating Deployments/version-control.md (hunk 6 above)
Comment (SergioQCL) [reply Cloud Application Manager/Automating Deployments/version-control.md]: Yes, I checked it and that was still happening. Should I create a ticket for this?

File: Cloud Application Manager/Automating Deployments/version-control.md (hunk 7 above)
Comment (SergioQCL) [reply Cloud Application Manager/Automating Deployments/version-control.md]: You already added it in a previous revision of the article. I'll move it before the picture to make it more visible if you agree.

File: Cloud Application Manager/Automating Deployments/version-control.md (hunk 6 above)
Comment (gsanchezu) [reply Cloud Application Manager/Automating Deployments/version-control.md]: Yes, please, and also add to the ticket that this doc should be updated, ;)

File: Cloud Application Manager/Automating Deployments/version-control.md (hunk 7 above)
Comment (gsanchezu) [reply Cloud Application Manager/Automating Deployments/version-control.md]: Yes, please :)

**Pull request merged**